aioesphomeapi/host_resolver.py

```diff
@@ -66,16 +67,16 @@ async def _async_zeroconf_get_service_info(
             f"Cannot start mDNS sockets: {exc}, is this a docker container without "
             "host network mode?"
```

Comment:
We need to move the creation of the instance to the resolve step so we don't make multiple since multiple names are supported